bin.src/validateDrp.py

```diff
@@ -29,37 +30,46 @@
 
 
 if __name__ == "__main__":
-    helpMessage = """Usage: validateDrp.py repo configFile
+    description="""
+    Calculate and plot validation Key Project Metrics from the LSST SRD.
 
-    Arguments
-    ---------
-    `repo` : str
-        path to a repository containing the output of processCcd
-    `configFile` : str
-        YAML configuration file declaring the parameters for this run.
-
-    Output
-    ------
+    Produces results to:
     STDOUT
         Summary of key metrics
     *.png
         Plots of key metrics.  Generated in current working directory.
+    *.json
+        JSON serialization of each KPM.
     """
-    if len(sys.argv) < 2:
-        print(helpMessage)
-        sys.exit(1)
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('repo', type=str, 
+                        help='path to a repository containing the output of processCcd')
+    parser.add_argument('--configFile', '-c', type=str, default=None,
+                        help='YAML configuration file validation parameters and dataIds.')
+    parser.add_argument('--verbose', '-v', default=False, action='store_true',
+                        help='Display additional information about the analysis.')
+    
+    args = parser.parse_args()
 
-    repo = sys.argv[1]
-    if not os.path.isdir(repo):
-        print("Could not find repo %r" % (repo,))
+    if not os.path.isdir(args.repo):
+        print("Could not find repo %r" % (args.repo,))
         sys.exit(1)
 
-    configFile = sys.argv[2]
+    kwargs = {}
```

Comment:
Looks like the commit which moved this line -- 25f714415 -- should be squashed into c8a608a7, which caused the original breakage. (If that's not appropriate for some reason, please fix the commit message -- "yes parameters" took me a while to parse.)